src/main/webapp/data/userMapData.json

```diff
@@ -290,9 +299,11 @@
       "Shanghai Guanghua Pudong Campus",
       "Shanghai University of Int. Business and Economics",
       "Tianfu University Chengdu China",
-      "Xi'an Jiaotong-Liverpool University"
+      "Xi'an Jiaotong-Liverpool University",
```

Comment:
Unnecessary change as you have deleted then again added it

Reply:
Hi @shaharyar-shamshi  , As you can see comma was not there so i added it.

Reply:
Good job I will review at night bit busy right now

Reply:
Hi @shaharyar-shamshi  , Kindly review when you have some time.